uweb3/templateparser.py

```diff
@@ -617,6 +646,14 @@ def Parse(self, **kwds):
     """
     self.ReloadIfModified()
     try:
+      if self._template_alias:
+        for key, value in self._replacements.items():
+          try:
+            x = TemplateTag.FromString(value) # XXX: This can probably be done in a cleaner way. How can we determine if a passed value is a tag?
```

Comment:
Maybe just TemplateString the value, and see if it parses into anything? That would allow both strings, tags, inlie templates (inception), and strings which contain tags etc.

Reply:
Hm,  doing it with a stringtemplate on the values would possible open us up to injection though, to avoid that we could incorporate a typed safestring which only does parsing on stings that have been escaped, or have been created from tempates, and not for example database output or other regular strings.